lib/npg_warehouse/loader/pacbio/run.pm

```diff
@@ -51,6 +51,12 @@ has 'run_uuid' =>
    required      => 1,
    documentation => 'The PacBio run unique identifier',);
 
+has 'id_script' =>
+  (isa           => 'Str',
+   is            => 'ro',
+   required      => 1,
+   documentation => 'A path to the script used to generate product ids',);
+
```

Comment:
The convention is to rely on executable scripts being on the PATH, no need to pass around a path to the script.